backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/memory/MemoryStorageHandler.java

```diff
@@ -44,12 +44,16 @@ public class MemoryStorageHandler {
      * @param vmForLogging
      *           The VM which the memory volumes being created belong to.<br/>
      *           Note: This parameter is used for logging purposed only.
+     * @param userId
+     *           The user doing the action.
+     * @param isFiltered
+     *           If the storage domains need to be filtered based on the user permissions.
```

Comment:
if it never makes sense to use a SD that the user doesn't have permissions on then there's no point in having this (isFiltered) argument